ci-operator/config/openshift/csi-operator/openshift-csi-operator-release-4.19.yaml

```diff
@@ -308,7 +320,7 @@ tests:
     workflow: openshift-e2e-azure-csi-extended
 - as: e2e-openstack
   optional: true
-  run_if_changed: ^(Dockerfile\.openstack-cinder|legacy/openstack-cinder-csi-driver-operator/.*)
+  run_if_changed: ^(Dockerfile\.openstack-cinder|legacy/openstack-cinder-csi-driver-operator/.*|Dockerfile\.openstack-manila|legacy/csi-driver-manila-operator/.*)
   steps:
```

Comment:
I realized that path separator needs to be escaped. We should escaped them before as well.

Reply:
Just pushed. Is that what you had in mind?

Reply:
No. Sorry - I meant - `^(Dockerfile\.openstack-cinder|legacy\/openstack-cinder-csi-driver-operator\/.*|Dockerfile\.openstack-manila|legacy\/csi-driver-manila-operator\/.*)`